build.gradle

```diff
@@ -11,7 +11,7 @@ plugins {
 defaultTasks 'licenseFormat', 'build'
 
```

Comment:
On line 8, you should update to the latest version of the SpongeGradle plugin: 0.8.1

`id 'org.spongepowered.plugin' version '0.8.1'`

I believe this adds the Sponge Maven repository. So you could remove line 23, but if not at least replace `http:` with `https: